CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 #### Upcoming Changes
 
+* feat(BREAKING): Add mod builtin [#1673](https://github.com/lambdaclass/cairo-vm/pull/1673)
+
+  Main Changes:
+  * Add the new `ModBuiltinRunner`, implementing the builtins `add_mod` & `mul_mod`
+  * Add temporary feature `mod_builtin` which adds `add_mod` & `mul_mod` to the `all_cairo` & `all_solidity` layouts so that the new builtins may be used in them. This feature will be removed once these builtins are properly added to the correct layouts
+  * Add method `VirtualMachine::fill_memory` in order to perform the new builtin's main logic from within hints
+  * Add hints to run arithmetic circuits using `add_mod` and/or `mul_mod` builtins
+
+  Other Changes:
+  * BREAKING: BuiltinRunner method signature change from
+  `air_private_input(&self, memory: &Memory) -> Vec<PrivateInput>` to `pub fn air_private_input(&self, segments: &MemorySegmentManager) -> Vec<PrivateInput>`
+  * Add `MayleRelocatable::sub_usize`
```

Comment:
Why not just `Sub<usize> for MaybeRelocatable`?

Reply:
I followed the existing convention for MaybeRelocatable operations, this should be refactored for all of the methods at once

Reply:
Sounds good.